integration/activemq-vertx-integration/src/main/java/org/apache/activemq/artemis/integration/vertx/ActiveMQVertxLogger.java

```diff
@@ -38,7 +38,7 @@
  * so an INFO message would be 191000 to 191999
  */
 @MessageLogger(projectCode = "AMQ")
-public interface ActiveMQVertxLogger extends BasicLogger {
+interface ActiveMQVertxLogger extends BasicLogger {
```

Comment:
If we create separate packages within the module, you won't be able to reuse the package private Logger.

Just because something is not using it now outside of the packet it doesn't mean it eventually couldn't happen IMO